### PR TITLE
[TST] Refactoring CircleCI setup

### LINKED ISSUE #1090: CI: Tests failing on ds005
Is this just a case of the cache needing bumping? Merged that last PR before I realized current master is failing.

### PR DESCRIPTION
**Pull request by oesteban**
Moves `--use-aroma` to ds054 with two ideas:
  * Shorten ds005 which was growing too large (CIFTI, partial exec, etc.)
  * Remove SyN from ds054 because it has fieldmaps, and make time for AROMA.

Overall, this PR should shorten the runtime of tests.

Also, figures/ folder has been pruned out of the outputs check (reports can be visualized and this check is a source of many errors).

When finished, this should fix #1090.

### CODE REVIEW

**Comment by oesteban** [comment]
Something weird is happening. Moving AROMA to ds054 now gives:

```
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/multiproc.py", line 68, in run_node
    result['result'] = node.run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 480, in run
    result = self._run_interface(execute=True)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 564, in _run_interface
    return self._run_command(execute)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 644, in _run_command
    result = self._interface.run(cwd=outdir)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base/core.py", line 520, in run
    runtime = self._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/interfaces/report_base.py", line 51, in _run_interface
    ReportCapableInterface, self)._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base/core.py", line 1020, in _run_interface
    self.raise_exception(runtime)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base/core.py", line 957, in raise_exception
    ).format(**runtime.dictcopy()))
RuntimeError: Command:
ICA_AROMA.py -den nonaggr -i /scratch/fmriprep_wf/single_subject_100185_wf/func_preproc_task_machinegame_run_01_wf/ica_aroma_wf/smooth/vol0000_xform-00000_merged_smooth.nii.gz -meldir /scratch/fmriprep_wf/single_subject_100185_wf/func_preproc_task_machinegame_run_01_wf/ica_aroma_wf/melodic -mc /scratch/fmriprep_wf/single_subject_100185_wf/func_preproc_task_machinegame_run_01_wf/bold_hmc_wf/normalize_motion/motion_params.txt -o out
Standard output:

------------------------------- RUNNING ICA-AROMA ------------------------------- 
--------------- 'ICA-based Automatic Removal Of Motion Artifacts' --------------- 

Warning! Please check whether the determined TR (of 1.0s) is correct!

Step 1) MELODIC
  - The specified MELODIC directory does not contain the required files to run ICA-AROMA. MELODIC will be run seperately.
Standard error:



An exception has been thrown
Logic error:- detected by Newmat: index error: requested index = 2

MatrixType = Rect   # Rows = 1; # Cols = 1

Traceback (most recent call last):
  File "/opt/ICA-AROMA/ICA_AROMA.py", line 193, in <module>
    aromafunc.runICA(fslDir, inFile, outDir, melDir, mask, dim, TR)
  File "/opt/ICA-AROMA/ICA_AROMA_functions.py", line 94, in runICA
    nrICs = int(float(subprocess.getoutput(cmd)))
ValueError: could not convert string to float: 'ERROR: Could not open file'
Return code: 1
```

And the weird issue of work/ subfolders not being stored as artifacts is replicated.

**Comment by effigies** [comment]
This looks reasonable, but I guess we'll see when tests finish.

File: fmriprep/interfaces/bids.py [hunk 1]
```diff
@@ -250,7 +254,10 @@ def _run_interface(self, runtime):
             if isdefined(self.inputs.extra_values):
                 out_file = out_file.format(extra_value=self.inputs.extra_values[i])
             self._results['out_file'].append(out_file)
-            _copy_any(fname, out_file)
+            self._results['compression'].append(_copy_any(fname, out_file))
+
+        if len(self._results['compression']) == 1:
+            self._results['compression'] = self._results['compression'][0]
```
Comment (effigies) [inline fmriprep/interfaces/bids.py]: I guess this is over here, now. So I'll just re-note that you can drop these two lines.

**Comment by oesteban** [comment]
Continued in #1098. Fix for #1090 will be addressed in a new PR.

**Pull request closed without merging**